debug_toolbar/apps.py

```diff
@@ -174,3 +175,37 @@ def check_panels(app_configs, **kwargs):
             )
         )
     return errors
+
+
+@register()
+def js_mimetype_check(app_configs, **kwargs):
+    """
+    Check that JavaScript files are resolving to the correct content type.
+    """
+    javascript_types = {
+        # Macs
+        "application/javascript",
+        # Windows
+        "text/javascript",
+    }
```

Comment:
Maybe this could be made less specific, something like:

```suggestion
    # Those types are fine. Different OSes return different types.
    javascript_types = {"application/javascript", "text/javascript"}
```

I'm not sure if that would address your concerns @tim-schilling ?